tracer/reporter.go

```diff
@@ -0,0 +1,92 @@
+package tracer
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	wf "github.com/wavefronthq/wavefront-sdk-go/senders"
+)
+
+type ApplicationTags struct {
```

Comment:
This should ideally be under `wavefront-sdk-go`. We will be leveraging this across several Go SDKs we build.